### PR TITLE
OLS-514: Make data collector configurable via olsconfig.yaml

### PR DESCRIPTION
**Pull request by onmete**
## Description

Make data collector configurable via olsconfig.yaml.

Tests should work after - https://github.com/openshift/lightspeed-operator/pull/324

## Type of change

- [x] Refactor
- [x] New feature

### CODE REVIEW

File: ols/app/models/config.py [hunk 1]
```diff
@@ -90,6 +91,21 @@ def _file_check(path: FilePath, desc: str) -> None:
         raise InvalidConfigurationError(f"{desc} '{path}' is not readable")
 
 
+def _get_log_level(value: str) -> int:
```
Comment (tisnik) [inline ols/app/models/config.py:94]: -> utils/logging.py perhaps?

Comment (onmete) [reply ols/app/models/config.py:94]: cause circular import - InvalidConfigurationError in .../config.py <-> _get_log_level in .../logging.py

**Comment by onmete** [comment]
Need to wait for operator release.

/hold

**Review by tisnik — COMMENTED** [review]
looks ok-ish, but please squast commits first

**Comment by tisnik** [comment]
/override "Red Hat Konflux / ols-e2e-tests-rosa / lightspeed-service"
/override "Red Hat Konflux / ols-enterprise-contract / lightspeed-service"

**Pull request merged**